File: app/src/main/java/com/mobility/myapplication/model/Dob.java
package com.mobility.myapplication.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created By JISHAN ANSARI On 10/25/2019
 */

public class Dob {


    @SerializedName("age")
    @Expose
    private Integer age;

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

}

File: app/src/main/java/com/mobility/myapplication/model/Dob.kt
package com.mobility.myapplication.model

import androidx.room.Entity
import androidx.room.ForeignKey
import androidx.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created By JISHAN ANSARI On 10/25/2019
 */
@Entity(
    tableName = "dob_table",
    foreignKeys = [ForeignKey(
        entity = Results::class,
        parentColumns = ["result_id"],
        childColumns = ["dob_id"],
        onDelete = ForeignKey.CASCADE
    )]
)
data class Dob(
    @PrimaryKey(autoGenerate = true)
    var dob_id: Int = 0,
    @SerializedName("age")
    @Expose
    var age: Int? = null
)